src/codec/builtin/FixSizedTypesCodec.ts

```diff
@@ -18,6 +18,36 @@ import * as Long from 'long';
 import {BitsUtil} from '../../util/BitsUtil';
 import {UUID} from '../../core/UUID';
 
+// Taken from long.js, https://github.com/dcodeIO/long.js/blob/master/src/long.js
+
+/**
+ * @type {number}
+ * @const
+ * @inner
+ */
+var TWO_PWR_16_DBL = 1 << 16;
```

Comment:
Please use `const` instead of `var` here and in the following constants.

Reply:
ok